staging/src/k8s.io/cloud-provider/controllers/service/controller_test.go

```diff
@@ -113,7 +113,7 @@ func tweakAddFinalizers(finalizers ...string) serviceTweak {
 
 func tweakAddDeletionTimestamp(time time.Time) serviceTweak {
 	return func(s *v1.Service) {
-		s.ObjectMeta.DeletionTimestamp.Time = time
+		s.ObjectMeta.DeletionTimestamp = &metav1.Time{Time: time,}
```

Comment:
This should be back-merged into the commit that added it

Reply:
ok, I tried to merge before, but found that "cleanup whitespaces" commit patch could not be merged directly after change, so I just added a new commit for simplicity.